packages/module/src/InvalidObject/InvalidObject.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Title } from '@patternfly/react-core';
+
+import Icon404 from './icon-404';
+import React from 'react';
+import classNames from 'classnames';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  "ins-c-component_invalid-component": {
+    display: "flex",
+    justifyContent: "center",
+    flexDirection: "column",
+    alignItems: "center",
+    textAlign: "center",
+    h1: {marginBottom: "30px"},
+    svg: {marginBottom: "30px"},
+    button: {fontSize: "20px"}
+  },
+  "ins-c-text__sorry": {maxWidth: "70%"},
+})
+
+// Don't use chrome here because the 404 page on landing does not use chrome
+const isBeta = () => {
+  return window.location.pathname.split('/')[1] === 'beta' ? '/beta' : '';
+};
+
+
+const InvalidObject: React.FunctionComponent = ({ ...props }) => {
+  const classes = useStyles();
+  const invalidObjectClasses = classNames("pf-v5-l-page__main-section", "pf-v5-c-page__main-section", classes['ins-c-component_invalid-component'])
+  return (
+    <section {...props} className={invalidObjectClasses}>
+      <Title headingLevel="h1" size="3xl">
+        We lost that page
+      </Title>
+      <Icon404 />
+      <Title headingLevel="h1" size="xl" className={classes['ins-c-text__sorry']}>
+        Let&apos;s find you a new one. Try a new search or return home.
+      </Title>
+      <Button variant="link" component="a" href={`${window.location.origin}${isBeta()}`}>
```

Comment:
The link should be a prop with a `/` fallback.

Reply:
I agree this should be a prop but adding props is going to break all the existing instances.

Reply:
@InsaneZein let's do breaking changes if necessary. When apps are migrating to our new repo, they need to expect some changes. There probably won't be a better opportunity to make the changes than now. Just we need to prepare a wrapper in FEC after this PR is merged which will use this component and make its new API compatible with the old one to ensure temporary compatibility.

Reply:
Ok! Sounds good to me. I'll make the changes. I'll probably need some help with the wrapper if possible.

Reply:
Sure, let's clean everything that should be cleaned. We can deal with the compatibility afterwards - a similar wrapper is here https://github.com/RedHatInsights/frontend-components/pull/1764/files#diff-de3a34ca94b2dff88e7502e7f8147238c7e316270e108643e104cbee1af26cb3

that one was to add a Red Hat specific description which we don't want in PatternFly extension

Reply:
The components are moving to a different repository. It is expected to make breaking changes. This library is not a continuation of FEC, but a generalization of the components in FEC and detaching any relation to HCC to allow the usage outside of HCC.